### PR TITLE
Feature be enj 28

### PR DESCRIPTION
**Pull request by dhruti96**
This is the API test script for the ENJ 28 Backend story for bulk question upload.

### CODE REVIEW

File: .idea/encodings.xml
Comment (ksharmatechie) [inline .idea/encodings.xml:1]: We never commit codeStyles in our commit. revert these commits.

File: target/test-classes/application.properties [hunk 1]
```diff
@@ -0,0 +1,24 @@
+# Application properties
```
Comment (ksharmatechie) [inline target/test-classes/application.properties]: We never commit. target folder in the commit. Target folder gets generated every time we do mvn clean install. So anything that gets generated need not be committed. It unnecessarily will occupy space on server

File: .idea/encodings.xml
Comment (dhruti96) [reply .idea/encodings.xml:1]: Deleted .idea and target folder

File: target/test-classes/application.properties (hunk 1 above)
Comment (dhruti96) [reply target/test-classes/application.properties]: Remove the target folder

**Review by ksharmatechie — APPROVED** [review]

**Pull request merged**